mypy/semanal_main.py

```diff
@@ -179,7 +179,7 @@ def process_top_levels(graph: Graph, scc: list[str], patches: Patches) -> None:
 
     # Reverse order of the scc so the first modules in the original list will be
     # be processed first. This helps with performance.
-    scc = list(reversed(scc))
+    scc = list(reversed(scc))  # noqa: FURB187
```

Comment:
This was a false positive, as this copy is intentional. Otherwise, it mutates the order of the scc args to the function, which breaks things.